README.adoc

```diff
@@ -1,6 +1,6 @@
 ---
 tags: []
-projects: []
+projects: [spring-cloud]
 ---
 :spring_version: current
 :spring_boot_version: 1.5.3.RELEASE
```

Comment:
I defer to @gregturn . I'm not sure what the impact of changing this is, and it was in the template already.